src/Sylius/Behat/Context/Setup/PromotionContext.php

```diff
@@ -262,9 +289,28 @@ private function configureActionTaxonFilter(ActionInterface $action, array $taxo
      * @param int $discount
      * @param array $configuration
      */
-    private function createPromotionWithPriceRangeFilter(PromotionInterface $promotion, $discount, array $configuration)
+    private function createFixedPromotionWithPriceRangeFilter(PromotionInterface $promotion, $discount, array $configuration)
+    {
+        $this->persistPromotionWithAction($promotion, $this->actionFactory->createItemFixedDiscount($discount), $configuration);
+    }
+
+    /**
+     * @param PromotionInterface $promotion
+     * @param int $discount
+     * @param array $configuration
+     */
+    private function createPercentagePromotionWithPriceRangeFilter(PromotionInterface $promotion, $discount, array $configuration)
```

Comment:
Anyway, Is there any need for such method? We reduce this call `$this->actionFactory->createItemPercentageDiscount($discount)` and put whole new method for this. I think it is not worth it.